release/update-rsync.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+#
+# Update rsync server directories with latest bundled source code
+# This will update rsync to the latest snapshot OR release tarball
+# available in the www service
+#
+
+test -e ~/.server.config && . ~/.server.config || exit 1
+
+cd `dirname $0`/..
+
+for version in `ls -1 $SQUID_VCS_PATH | grep squid | cut -d- -f2`; do
+	rm -rf tmp
```

Comment:
FWIW, I think it is too dangerous to forcefully delete directories with such common names as `tmp`. I know I would be rather upset if this script removed my custom `tmp` directory (I do have one), even though it was named `tmp`. AFAICT, the actual name is not important to this script, so I suggest using something more likely to be unique (e.g., `update-rsync-tmp`).

Reply:
Bug fixes welcome in followup PRs. This one is scoped at importing the current status-quo automation.